src/common/utils/redis/helper.go

```diff
@@ -171,24 +175,18 @@ func DefaultPool() *redis.Pool {
 			idleTimeout = poolIdleTimeout
 		}
 
-		pool = &redis.Pool{
-			Dial: func() (redis.Conn, error) {
-				url := config.GetRedisOfRegURL()
-				if url == "" {
-					url = "redis://localhost:6379/1"
-				}
-
-				return redis.DialURL(url)
-			},
-			TestOnBorrow: func(c redis.Conn, t time.Time) error {
-				_, err := c.Do("PING")
-				return err
-			},
-			MaxIdle:     maxIdle,
-			MaxActive:   maxActive,
-			IdleTimeout: time.Duration(idleTimeout) * time.Second,
-			Wait:        true,
+		url := config.GetRedisOfRegURL()
```

Comment:
Is this `config.GetRedisOfRegURL` still match the configurations? Will it be sentinel URL of Redis?

Can you move this function to `lib/redis`?

Reply:
If this function moved to `lib/redis`, this pkg `utils/redis` can be removed.  

`RequireLock` in this pkg is not used by others and can be removed.